cartography/intel/aws/ecr_image_layers.py

```diff
@@ -721,30 +729,73 @@ def sync(
             current_aws_account_id,
         )
 
-        # Get ECR images from graph using standard client function
-        from cartography.client.aws.ecr import get_ecr_images
+        # Query for ECR images with all their existing properties to preserve during layer sync
```

Comment:
Can no longer use the ecr client defined in `cartography.client.aws.ecr.get_ecr_images`